lib/squadster/workers/notify_attendants.ex

```diff
@@ -1,11 +1,39 @@
 defmodule Squadster.Workers.NotifyAttendants do
   use Task
 
+  import Ecto.Query
+  import HTTPoison
+
+  alias Squadster.Formations.SquadMember
+  alias Squadster.Repo
+
+  @bot_endpoint "https://mercury-dev.herokuapp.com/message"
+  @message "Завтра вы ответственный за лопату!"
+  @request_headers [{"content-type", "application/json"}]
+
   def start_link(args) do
     Task.start_link(__MODULE__, :run, [args])
   end
 
   def run(_args) do
-    # TODO: implement
+    next_day_duties = from(member in SquadMember, where: member.queue_number == 1)
```

Comment:
It should also select by class day (tomorrow). Day of the week can be taken from Timex (see `Helpers.Dates`), just map it to class_day